chart/openfaas/templates/alertmanager-dep.yaml

```diff
@@ -16,6 +16,7 @@ spec:
       labels:
         app: alertmanager
       annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
Just curious.. why doesn't this work for AlertManager? Because of its HA discovery mechanism?